### ISSUE TITLE
Shariant import - wrong "source URL"

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#653]
[SA Path classification](http://frgeneseq02.imvs.sa.gov.au:92/classification/classification/35832)

Has "Source URL" set to https://variantgrid.com/classification/classification/19500

While it should really be from shariant.org.au

**Comment by TheMadBug** [comment]
I'll look into developing a script to fix this (want to just update the data in place rather than patch the data).

**Comment by davmlaw** [comment]
I don't get how this happened, running on SA Path server:

```
from library.oauth import OAuthConnector
from sync.models.models import SyncDestination

sync_destination = SyncDestination.objects.get(name='shariant_download')
config = sync_destination.config
shariant = OAuthConnector.shariant_oauth_connector(config["sync_details"])
source_url = shariant.url(f'classification/classification/42')
In [13]: source_url                                                                                                       
Out[13]: 'https://shariant.org.au/classification/classification/42'
```

**Comment by davmlaw** [comment]
The full entry for:

```c.evidence["source_url"]```

looks like:

```
{'note': 'Unable to import vcgs:0207, vcgs:proband, vcgs:noncoding, import_warnings, vcgs:nmd_benign, vcgs:comparable_benign, vcgs:disease_mechanism, vcgs:alternative_change',
 'value': 'https://variantgrid.com/classification/classification/19500',
 'db_refs': [{'db': 'HTTPS',
   'id': 'HTTPS: //variantgrid.com/classification/classification/19500',
   'idx': '//variantgrid.com/classification/classification/19500',
   'url': 'https://variantgrid.com/classification/classification/19500'}],
 'immutable': 'api'}
```

Spent a bit more time following the flow of code, still don't get how this was set

**Comment by davmlaw** [comment]
@TheMadBug do you have any idea how this could have happened? I'm stumped

**Comment by TheMadBug** [comment]
We can at least ignore the DB refs, they're just generated because the code saw a URL in value.

But how it came to be variantgrid.com and not shariant.org after clearly downloading it from shariant...

Do you know if all new records were assigned variantgrid.org or just some of them?

**Comment by davmlaw** [comment]
I looked and couldn't find any... then noticed the original example has now been fixed.

Looking at the history, it appears that it's switching back and forth between shariant and variantgrid!

http://frgeneseq02.imvs.sa.gov.au:92/classification/classification/35832/history

**Comment by davmlaw** [comment]
This is def not because the same classification is coming through both systems - NM_000132.3(F8):c.2163G>T doesn't even exist on vg.com let alone have a classification

**Comment by davmlaw** [comment]
This was caused by leaving the code that loaded data from a JSON file rather than the connection. Thus it was cycled back and forth between VG/Shariant as the automated syncs ran.

I have reverted that code now that everything is set to Shariant. Also restarted DB workers service to make sure new code is loaded

Will check in 1 day to make sure issue doesn't reoccur - check history link above and also check in shell

```
from classification.models import Classification

vg_com_qs = Classification.objects.filter(evidence__source_url__value__startswith='https://variantgrid.com')
bad_qs = vg_com_qs.exclude(evidence__curation_system__value='VariantGrid')
```
